scripts/build-k8s-on-amd64/Dockerfile.arm64

```diff
@@ -24,7 +24,9 @@ ENV GOARM=6 \
     FLANNEL_VERSION=v0.5.5 \
     K8S_VERSION=v1.2.0 \
     REGISTRY_VERSION=v2.3.1 \
-    KUBE_BUILD_PLATFORMS="linux/arm64"
+    KUBE_BUILD_PLATFORMS="linux/arm64" \
+    TAG=0.61 \
```

Comment:
for example `Dockerfile` line 83:
```
    && make server \
```
in `exec-healthz/Makefile`:
```
ifndef TAG
    $(error TAG is undefined)
endif
```
the build fails without the TAG env